app/views/Partners/PartnersOverview.tsx

```diff
@@ -113,6 +130,24 @@ const PartnersScreen = ({ navigation }: PartnersScreenProps): JSX.Element => {
         <View style={styles.divider} />
         <View style={{ height: 24 }} />
       </ScrollView>
+      <View style={styles.bottomSheet}>
```

Comment:
Can we keep this hidden or removed? We still don't want this visible to the end user.

Reply:
sure thing!